.travis.yml

```diff
@@ -20,7 +19,7 @@ php:
 matrix:
   fast_finish: true
   include:
-    - php: 5.3
+    - php: 5.4
```

Comment:
Let's revert the changes for the PHP version right now. We can consider to do that later.